api/room/schema_query.py

```diff
@@ -167,15 +170,16 @@ def resolve_all_rooms(self, info, **kwargs):
 
     @Auth.user_roles('Admin')
     def resolve_all_remote_rooms(self, info):
-        service = Credentials().set_api_credentials()
         page_token = None
+        switcher = map_remote_room_location_to_filter()
+        location = get_user_from_db().location
         remote_rooms = []
         while True:
-            calendar_list = service.calendarList().list(
-                pageToken=page_token).execute()
+            calendar_list = CalendarApi().calendar_list(page_token)
```

Comment:
Suppose the calendar API is down or there are no resources, how have you handled this possible exception?

Reply:
Work on the codeclimate error

Reply:
> Suppose the calendar API is down or there are no resources, how have you handled this possible exception?

I have handled exceptions that might be raised by the calendar API. Non-existence of resources on the other hand isn't an exception, in this case, an empty list will be returned.

Reply:
> Work on the codeclimate error

This has been solved.